packages/testsweets/lib/src/models/application_models.dart

```diff
@@ -15,47 +15,37 @@ class WidgetDescription with _$WidgetDescription {
     String? id,
 
     /// The name of the view this widget was captured on
-    required String viewName,
+    String? viewName,
 
     /// The orignal name of the view this widget was captured on before the prettify
-    required String originalViewName,
+    String? originalViewName,
 
     /// The name we want to use when referring to the widget in the scripts
-    required String name,
+    @Default('') String name,
```

Comment:
This is also required. I don't know why all these values are now nullable. Either something went wrong or we've made it so that widgets can get captured without a name and without belonging to a view.

Reply:
I was able to remove repeating calls where I just add an empty string as an initial value

Reply:
I'm happy with this one